Packs/CommonScripts/Scripts/ClearUserSession/ClearUserSession.py

```diff
@@ -285,29 +299,31 @@ def create_readable_output(outputs: list):
     Args:
         outputs (dict): A dictionary where each key is a username and the value is
             another dictionary containing:
-            - "Result" (str): The session result (e.g., Success, Failure).
-            - "Source" (str, optional): The brand(s) associated with the session.
             - "Message" (str): A detailed message related to the session status.
+            - "Result" (str): The session result (e.g., Success, Failure).
+            - "Brand" (str): The brand associated with the session.
+            - "UserName" (str): The user name a session cleared for.
 
     Returns:
         str: A markdown-formatted table summarizing user session statuses.
     """
     data_users_list = [
         {
-            "User Name": details["UserName"],
-            "Result": details["Result"],
-            "Source": ", ".join(details.get("Source", [])),
             "Message": details["Message"],
+            "Result": details["Result"],
+            "Brand": details["Brand"],
+            "UserName": details["UserName"],
         }
         for details in outputs
     ]
 
     readable_output = tableToMarkdown(
         name="User(s) Session Status",
         t=data_users_list,
-        headers=["User Name", "Result", "Source", "Message"],
+        headers=["Message", "Result", "Brand", "UserName"],
```

Comment:
I think the order of the headers in the md should stay the same - first username then message